packages/@icon-magic/cli/src/usage.ts

```diff
@@ -0,0 +1,62 @@
+const commandsDescription = {
+  build: {
+    name: 'build',
+    description:
+      'Construct flavors that an icon from its variants, after applying the build plugins.'
+  },
+  generate: {
+    name: 'generate',
+    description:
+      'Generates the flavors of the icon in the extension types that it can be consumed.'
+  },
+  distribute: {
+    name: 'distribute',
+    description:
+      'Moves an icon from the source to the destination, applying plugins if specified.'
+  }
+};
+
+const options = {
+  config: {
+    name: '-c, --config',
+    description: 'Path to config file name'
+  },
+  inputPath: {
+    name: '-i, --inputPath',
+    description: 'Path to the input directory of icons'
+  },
+  outputPath: {
+    name: '-o, --outputPath',
+    description:
+      'Path to the output directory where the generated assets are to be written to'
+  },
+  help: {
+    name: '-h, --help',
+    description: 'Display usage'
+  },
+  version: {
+    name: '-v, --version',
+    description: 'Display version'
+  }
+};
+
+function getDescription(descriptions: {}) {
+  return Object.values(descriptions)
+    .map(function(obj) {
+      return `  ${obj.name}${getSpaces(obj.name)}${obj.description}`;
+    })
+    .join('\n');
+}
+
+function getSpaces(word: string): string {
+  const COLUMN_WIDTH = 20;
+  return ' '.repeat(COLUMN_WIDTH - word.length);
+}
+
+const usage = `Usage: \n  icon-magic [command] [options] <directories ...>\n
+Commands: \n${getDescription(commandsDescription)}\n
+Options: \n${getDescription(options)}\n`;
+
+export function printUsage() {
+  console.log(usage);
```

Comment:
Commander gives you usage output for free: https://www.npmjs.com/package/commander#automated---help (and sections thereafter)

Reply:
Nice! Integrated.